packages/i18n/src/locales/en_US.ts

```diff
@@ -1979,6 +1979,16 @@ const translations: Catalog = {
               description: 'Adds an removeOne to the bulk operation.',
               example: 'bulkOp.find(...).removeOne()',
             },
+            'delete': {
+              link: 'https://docs.mongodb.com/manual/reference/method/Bulk.find.delete/',
```

Comment:
Do we expect the links to work at some point in the future? Right now the `delete` and `deleteOne` links give a 404 error unfortunately (since they link to the old shell's API methods).

Reply:
Yes, I’d expect that they work once the documented for the linked ticket here is finished up